common/exception/ErrorMessage.java

```diff
@@ -228,6 +232,8 @@ public static void loadConstants() {
                 new Transaction(16, "Could not acquire lock for data transaction. A schema session may have been left open.");
         public static final Transaction RPC_PREFETCH_SIZE_TOO_SMALL =
                 new Transaction(17, "RPC answer streaming prefetch size must be at least 1, is set to: %d.");
+        public static final Transaction TRANSACTION_TIMEOUT_NOT_CONFIGURABLE =
+                new Transaction(18, "Transaction timeout cannot be configured at the '%s' level.");
```

Comment:
added a new error if someone tries to set a transaction time at below-transaction level, which is nonsense